app/views/reviews/_form.html.haml

```diff
@@ -2,14 +2,14 @@
   %h1 Edit Review
 
 = form_for [@piece, @review] do |f|
-  %div
+  %div.review-form-title
```

Comment:
`%div.review-form-title` can be written as `.review-form-title` since `%div` is implicit